src/preprocessor.js

```diff
@@ -0,0 +1,49 @@
+import { templateResource } from "./templates";
+import { expressResource } from "./expressions";
+
+/**
+ * @param {Array<Objects>} persona 
+ * @returns {Array<Objects} the parsed persona reiterated by the chain of callbacks
+ */
+export const parsePersona = persona => iteratePersona(iteratePersona(iteratePersona(persona, templateResource), expressResource), pathResource);
```

Comment:
Whoa! Let's think of a better way to write this. Would something like this work?
```
[template, express, path].reduce((accumulator, val) => {
  return iteratePersona(accumulator, val);
}, persona)
```
Or even one-line it...but I'd avoid trying to do too much in a single line.